membership-attribute-service/app/models/ContributionData.scala

```diff
@@ -17,7 +17,7 @@ object ContributionData {
     override def writes(o: ContributionData): JsValue = Json.obj(
       "created" -> o.created,
       "currency" -> o.currency.toString,
-      "amount" -> o.amount,
+      "price" -> o.amount,
```

Comment:
could keep this as `amount` as we use that convention in manage.theguardian.com for contributions

Reply:
I'd be happy for it to remain as is but for consistency it might make sense to change it?

Reply:
yep will keep it changed